darwin/DarwinProcess.c

```diff
@@ -223,6 +223,7 @@ void DarwinProcess_setFromKInfoProc(Process* proc, const struct kinfo_proc* ps,
       /* e_tdev = (major << 24) | (minor & 0xffffff) */
       /* e_tdev == -1 for "no device" */
       proc->tty_nr = ps->kp_eproc.e_tdev & 0xff; /* TODO tty_nr is unsigned */
+      proc->translated = ps->kp_proc.p_flag & P_TRANSLATED;
```

Comment:
When was the `P_TRANSLATED` flag introduced in the headers?

Reply:
It is present since at least Mac OS X 10.5: https://opensource.apple.com/source/xnu/xnu-1228/bsd/sys/proc.h.auto.html (I suppose it was created for the first PPC/x86 Rosetta).